### DISCUSSION TITLE
fatal error: echo_canceller.h

### DISCUSSION THREAD

**Discussion by HighnessKun** [community/community#50434] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

An attempt to compile a tflite_micro_speech_demo routine using aos-tools fails with fatal error: echo_canceller.h: No such file or directory 33 | #include "echo_canceller.h"

**Comment by otter-computer** [comment]
Hi there,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
